### DISCUSSION TITLE
[A-L] (12 Gennaio 2022) Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#358] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152746151-f47ed153-f2ab-4426-9849-30c281d0350f.png)

**Comment by i0nut02** [comment]
è soddisfacibile per 
p = 0, q = 0, r = 0

**Comment by neoSnakex34** [comment]
![gitubs](https://user-images.githubusercontent.com/97003970/152767494-f280c2e5-aff6-47e8-832b-b499d32e5d23.jpg)
@i0nut02 @Exyss

**Comment by Exyss** [comment]
È soddisfacibile, ma il metodo della risoluzione applicato sopra è errato, poiché non si può risolvere una clausola con 2 literals diversi.
Inoltre, l'algoritmo afferma che nasce una qualsiasi scatola vuota, allora è insoddisfacibile. Dunque applicando la risoluzione errata la risposta in realtà dovrebbe essere unsat, non sat

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/210138971-e8f0b601-2711-4882-8716-da34a724c181.png)
